### DISCUSSION TITLE
config.yml 的内容是什么 我这边用docker部署的时候没有生成.config文件

### DISCUSSION THREAD

**Discussion by moyuanhua** [coder/code-server#2135] [Q&A]
config.yml 的内容是什么 我这边用docker部署的时候没有生成.config文件

**Comment by shuchenliu** [comment]
Hi @moyuanhua. 

依据 [FAQ](https://github.com/cdr/code-server/blob/master/doc/FAQ.md#how-does-the-config-file-work), `config.yaml` 的基本内容是
```yaml
bind-addr: 127.0.0.1:8080
auth: password
password: somepassword # This is randomly generated for each config.yaml
cert: false
```
而[安装指导](https://github.com/cdr/code-server/blob/v3.5.0/doc/install.md#docker)里描述，docker 会读取本地`$HOME/.config/code-server/config.json` 作为配置文件。


-------

Some additional notes:

A native Chinese speaker myself, I understand the struggle of describing a problem in a language that you might not be so familiar with when the problem itself seems annoying enough - but since GitHub is a used by developers around the world, I would say that it'd be courteous, and also for the sake of a speedy resolution of your issue, to maybe translate the question into English using tools like Google translate before putting it up, instead of assuming someone will just understand it or translate it for you, especially that your question may help others who encounter similar issues.

同为说中文的人，我能理解用不熟悉的语言描述一个复杂的问题不是那么容易。 只是GitHub是一个全球开发者使用的网站，我想出于礼节考虑，以及为了你的问题能够得到迅速解答，也许应该使用Google translate这样的工具在发布前将问题翻译成英文，而不是假定会有人能看懂问题或者将替你翻译， 特别是你的问题也可能帮助其他遭遇类似困难的人。
